cdap-ui/app/cdap/components/AbstractWidget/DLPCustomWidget/index.tsx

```diff
@@ -85,3 +87,11 @@ export default function DLPWidget(props) {
 }
 
 (DLPWidget as any).propTypes = WIDGET_PROPTYPES;
+(DLPWidget as any).getWidgetAttributes = () => {
+  return {
+    // TODO: Need to investigate 'transforms' more
+    // transforms: { type: 'ITransformProp[]', required: true },
```

Comment:
not necessary. We're not going to support DLP widget for the JSON widget creator

Reply:
Resolved